util/pyclient/test_client.py

```diff
@@ -95,33 +108,16 @@ def testTimeout(self):
         trio.run(self._testTimeout, write, False)
 
     async def _testTimeout(self, msg, read_only):
-       config = self.config._replace(req_timeout_milli=100)
+       config = self.config._replace(req_timeout_milli=0)
```

Comment:
what does a timeout of 0 mean?

Reply:
it means the client will fail immediately.
Notice that before the change, we didn't even start the servers on this test (the purpose is to test the client behavior)